cleanlab_studio/studio/studio.py

```diff
@@ -133,16 +142,18 @@ def apply_corrections(self, cleanset_id: str, dataset: Any) -> Any:
             new_labels = final.select(
                 [id_col, "action", "__cleanlab_final_label"]
             ).withColumnRenamed("__cleanlab_final_label", label_column)
-            corrected_df = (
-                dataset.drop(label_column)
+            return (
+                corrected_ds.drop(label_column)
                 .join(new_labels, on=id_col, how="right")
                 .where(new_labels["action"] != "exclude")
                 .drop("action")
             )
-            return corrected_df
-
         elif isinstance(dataset, pd.DataFrame):
```

Comment:
nit: this is separate to the changes you've made here, but can you add a default case to this conditional for if the dataset is not a pyspark or pandas dataframe? currently, we'll just return None -- we should instead raise a value error. would be good if this func was typed correctly as well (dataset should not be `Any`)